lib/Controller/ApiController.php

```diff
@@ -930,11 +933,59 @@ public function deleteOption(int $id): DataResponse {
 
 		$this->optionMapper->delete($option);
 
+		// Reorder the remaining options
+		$options = array_values($this->optionMapper->findByQuestion($question->getId()));
+		foreach ($options as $order => $option) {
+			$option->setOrder($order);
+			$this->optionMapper->update($option);
+		}
+
 		$this->formsService->setLastUpdatedTimestamp($form->getId());
 
 		return new DataResponse($id);
 	}
 
+	/**
+	 * Reorder options for a given question
+	 * @param int $id The question ID
+	 * @param int[] Order to use
```

Comment:
```suggestion
	 * @param int[] $order Order to use
```
Parameter name was missing